### ISSUE TITLE
BulkExecutor - Method not found

### ISSUE THREAD

**Issue by JordanMarr** [Azure/azure-cosmosdb-bulkexecutor-dotnet-getting-started#49]
Error:
Method not found: 'Void Microsoft.Azure.CosmosDB.BulkExecutor.BulkExecutor..ctor(Microsoft.Azure.Documents.Client.DocumentClient, Microsoft.Azure.Documents.DocumentCollection)'.

I have tried many different combinations of versions between "BulkExecutor" and "DocumentDB.Core", but none of them work.  What is the magic combination?

**Comment by andremaree** [comment]
Microsoft.Azure.DocumentDB= 2.1.3

**Comment by galmok** [comment]
Correct, BulkExecutor still requires an ancient version of DocumentDB.Core (2.1.3). I wish they would upgrade that dependency.

**Comment by ealsur** [comment]
Version 1.8.1 https://www.nuget.org/packages/Microsoft.Azure.CosmosDB.BulkExecutor/1.8.1 is on 2.5.1
